src/validation-failures/content-type-mismatch.ts

```diff
@@ -2,14 +2,13 @@ import {
   CONTENT_TYPE_MISMATCH_ERROR,
   CONTENT_TYPE_PREFIX,
 } from '../utilities/constants';
+import ValidationFailure from './validation-failure';
 
-class ContentTypeMismatchError extends TypeError {
+class ContentTypeMismatchError extends ValidationFailure {
```

Comment:
Should we keep 'Error' at the end of all these class names?

Reply:
Thanks for catching this! I meant to take them out, but i must have missed that in the repetition of changing all these.